frontend/app/components/wp-create/wp-create.controller.ts

```diff
@@ -97,7 +104,20 @@ export class WorkPackageCreateController {
             });
         }
       })
-      .catch(error => this.wpNotificationsService.handleErrorResponse(error));
+      .catch(error => {
+        if (error.errorIdentifier == "urn:openproject-org:api:v3:errors:MissingPermission") {
+          this.halRequest.get(this.v3Path.root()).then((root:HalResource) => {
```

Comment:
After merging #5219 you can now simply use the [root resource dm](https://github.com/opf/openproject/blob/dev/frontend/app/components/api/api-v3/hal-resource-dms/root-dm.service.ts#L38) like this (after importing):

```
this.RootDm.load().then((root:RootResource) => {
...
```